automatic/azcopy10/azcopy10.nuspec

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>azcopy10</id>
+    <version>10.21.1</version>
+    <title>azcopy10</title>
+    <authors>Microsoft</authors>
+    <owners>jbpaux, pauby</owners>
+    <projectUrl>https://github.com/Azure/azure-storage-azcopy</projectUrl>
+    <packageSourceUrl>https://github.com/jbpaux/chocolatey-packages/tree/master/azcopy10</packageSourceUrl>
```

Comment:
You should update this line to target path

Reply:
I've updated this.